tests/ctst/steps/iam-policies/common.ts

```diff
@@ -209,4 +210,14 @@ Then('the user should be able to perform successfully', function () {
     } else {
         assert.strictEqual(this.result?.statusCode, 200);
     }
-});
\ No newline at end of file
+});
+
+Then('the user should receive an access denied error', function () {
+    this.endForType();
+    if (this.saved.ifS3Standard) {
+        assert.strictEqual(this.result?.err?.includes("AccessDenied"), true);
+    } else {
+        assert.strictEqual(this.result?.statusCode, 403);
+        assert.strictEqual(this.result?.data?.includes("AccessDenied"), true);
+    }
+});
```

Comment:
Suggestion: make this function generic:

```suggestion
Then('the user should receive a {string} error', function (errorName) {
    this.endForType();
    if (!this.result) {
        throw new Error('Expecting a result after calling the API, but got undefined.');
    }
    if (this.saved.ifS3Standard) {
        assert.strictEqual(this.result?.err?.includes(errorName), true);
    } else {
        assert.strictEqual(this.result?.statusCode, 403);
        assert.strictEqual(this.result?.data?.includes(errorName), true);
    }
});
```

Note: I added a precheck for the `this.result` existence, in case something goes wrong, so that the test fails also in this case